acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/transaction/privacy/PrivacyRequestFactory.java

```diff
@@ -16,12 +16,43 @@
 
 import static java.util.Collections.singletonList;
 
+import org.hyperledger.besu.enclave.types.PrivacyGroup;
+import org.hyperledger.besu.ethereum.api.jsonrpc.internal.privacy.parameters.CreatePrivacyGroupParameter;
+
+import java.util.Collections;
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import org.web3j.protocol.Web3jService;
 import org.web3j.protocol.besu.Besu;
 import org.web3j.protocol.core.Request;
 import org.web3j.protocol.core.Response;
 
 public class PrivacyRequestFactory {
+
+  public static class GetPrivacyPrecompileAddressResponse extends Response<String> {}
+
+  public static class GetPrivateTransactionResponse extends Response<HashMap<String, String>> {
+    final String privateFrom;
+
+    @JsonCreator
+    public GetPrivateTransactionResponse(
```

Comment:
Bit torn here, the private transaction response contains a lot more than just result. Thinking we should probably define the response type correctly with all fields even if we are only checking the result at the moment.